### ISSUE TITLE
Exception while creating tooltip: MonoDevelop.SourceEditor.Wrappers.TooltipProviderWrapper

### ISSUE THREAD

**Issue by KirillOsenkov** [mono/monodevelop#5995]
VSMac 7.6.5.20

```
ERROR [2018-09-16 15:02:41Z]: -------- Exception while creating tooltip: MonoDevelop.SourceEditor.Wrappers.TooltipProviderWrapper
System.NullReferenceException: Object reference not set to an instance of an object
  at MonoDevelop.SourceEditor.SourceEditorView.MonoDevelop.Ide.Editor.ITextEditorImpl.GetEditorWindowOrigin () [0x00000] in /Users/vsts/agent/2.140.0/work/1/s/monodevelop/main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor/SourceEditorView.cs:3092 
  at MonoDevelop.Ide.Editor.TooltipProvider.ShowTooltipWindow (MonoDevelop.Ide.Editor.TextEditor editor, MonoDevelop.Components.Window tipWindow, MonoDevelop.Ide.Editor.TooltipItem item, Xwt.ModifierKeys modifierState, System.Int32 mouseX, System.Int32 mouseY) [0x0002f] in /Users/vsts/agent/2.140.0/work/1/s/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor/TooltipProvider.cs:157 
  at MonoDevelop.SourceEditor.Wrappers.TooltipProviderWrapper.ShowTooltipWindow (Mono.TextEditor.MonoTextEditor editor, Xwt.WindowFrame tipWindow, System.Int32 offset, Gdk.ModifierType modifierState, System.Int32 mouseX, System.Int32 mouseY, MonoDevelop.Ide.Editor.TooltipItem item) [0x0000c] in /Users/vsts/agent/2.140.0/work/1/s/monodevelop/main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.Wrappers/TooltipProviderWrapper.cs:147 
  at Mono.TextEditor.TextArea+<TooltipTimer>d__384.MoveNext () [0x00253] in /Users/vsts/agent/2.140.0/work/1/s/monodevelop/main/src/addins/MonoDevelop.SourceEditor2/Mono.TextEditor/Gui/TextArea.cs:3253 
```

> VS bug [#685790](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/685790)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.8.0.531 (master)

Pull Request #6002 merged by: Mike Krüger
Author: mono
Commit: 6fcd6df467261e993f35e0b9d2b72e96acd9f0f9 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.7.0.1379 (release-7.7)

Pull Request #6003 merged by: Lluis Sanchez
Author: mono
Commit: 62286f9ed8921a6ef666b6f317cd0d3e1b6071ba (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #6002 merged by: Mike Kr??ger
Author: mono
Commit: 6fcd6df467261e993f35e0b9d2b72e96acd9f0f9 (xamarin/monodevelop)
